### PR TITLE
feature (ref 11961): allow to delete Gislayer

### PR DESCRIPTION
**Pull request by jalalifardemos**
### Ticket: 
DPLAN-11961

Description: get access to delete GisLayer

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [x] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/ResourceTypes/GisLayerResourceType.php [hunk 1]
```diff
@@ -84,6 +84,11 @@ public function isAvailable(): bool
         return true;
     }
 
+    public function isDeleteAllowed(): bool
```
Comment (sierrazamudiodemos) [inline demosplan/DemosPlanCoreBundle/ResourceTypes/GisLayerResourceType.php:87]: Lets add the permission area_map_participation_area as it is done with the isGetAllowed, and isListAllowed, in this way, the deletion is also protected

**Review by sierrazamudiodemos — CHANGES_REQUESTED** [review]
Add area_map_participation_area permission to isDeleteAllowed (as it is done with isGetAllowed, and isListAllowed)

File: demosplan/DemosPlanCoreBundle/ResourceTypes/GisLayerResourceType.php (hunk 1 above)
Comment (jalalifardemos) [reply demosplan/DemosPlanCoreBundle/ResourceTypes/GisLayerResourceType.php:87]: done in [5250f95](https://github.com/demos-europe/demosplan-core/pull/3347/commits/5250f95a3e681ed932017b8b3b13dcb706a4156f)

**Review by sierrazamudiodemos — APPROVED** [review]
BE works. FE does not refresh directly

**Comment by sierrazamudiodemos** [comment]
@salisdemos the BE part is fixed, could you the FE part? it seems that the after deleting, the FE does not detect the changes directly, let us know if there is something to do in the BE side

**Review by sierrazamudiodemos — APPROVED** [review]
thanks florian and saeed for expanding the resource type :) it works

**Review by sakutademos — APPROVED** [review]
lgtm, code review

**Pull request merged**